Infrastructure-Scripts/Remove-AzSqlIpException.ps1

```diff
@@ -0,0 +1,51 @@
+<#
+    .SYNOPSIS
+    Remove the whitelisted IP from the firewall rule name
+    .DESCRIPTION
+    Remove the whitelisted IP from the firewall rule name
+    .PARAMETER Name
+    The name of the firewall rule
+    .PARAMETER ServerName
+    Name of the Sql server
+    .PARAMETER WhatsMyIpUrl
+    The url value, "https://ifconfig.me/ip"  that needs to get passed in to identify the IP Address
```

Comment:
```suggestion
    The url value, e.g. "https://ifconfig.me/ip" that is called to identify the IP Address of the machine running the script
```

Reply:
As the script name is generic, do we want this parameter to be part of a parameter set, e.g. 'DetectIPAddress'

and another parameter IPAddress in another parameter set e.g. 'ProvidedIPAddress'